roles/ceph-grafana/tasks/configure_grafana.yml

```
@@ -25,7 +25,10 @@
     path: "{{ item }}"
     state: directory
     recurse: yes
+    owner: '{{ grafana_uid }}'
```

just cosmetic change,

why don't we have double quotes everywhere?
```suggestion
    owner: "{{ grafana_uid }}"
```

just to keep consistency

---

:+1: